lib/asar.js

```diff
@@ -103,7 +104,9 @@ module.exports.createPackageFromFiles = function (src, dest, filenames, metadata
 
     console.log(`Ordering file has ${((total - missing) / total) * 100}% coverage.`)
   } else {
-    filenamesSorted = filenames
+    for (const file of filenames) {
+      filenamesSorted.push(file.startsWith(src) ? file : path.join(src, file))
```

Comment:
Hi, I'm removing this line since it is causing problems on Windows (https://github.com/electron/asar/issues/158).

Reply:
If someone wants to fix it the issue will be `/` and `\` (path separators)